networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc

```diff
@@ -15,10 +15,14 @@ This functionality is generally most useful for clusters installed on bare-metal
 
 include::modules/nw-externalip-about.adoc[leveloffset=+1]
 
+include::modules/nw-assign-ip.adoc[leveloffset=+1]
+
 include::modules/nw-externalip-object.adoc[leveloffset=+1]
 
 include::modules/nw-externalip-configuring.adoc[leveloffset=+1]
```

Comment:
Can this module, modules/nw-externalip-configuring.adoc[leveloffset=+1], be replaced by the new procedure?